tox.ini

```diff
@@ -4,6 +4,7 @@ envlist =
        {py310}-{django42,django50,django51,djangomain}
        {py311}-{django42,django50,django51,djangomain}
        {py312}-{django42,django50,django51,djangomain}
+       {py313}-{django51,djangomain}
```

Comment:
Python 3.13 will be supported only by Django 5.1 and above: https://forum.djangoproject.com/t/backport-python-3-13-support-in-django-5-0/34671/2